integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/blockprune/BlockPruneQueryTestCase.scala

```diff
@@ -91,6 +121,46 @@ class BlockPruneQueryTestCase extends QueryTest with BeforeAndAfterAll {
       Seq(Row("b", 240001)))
   }
 
+  test("test block prune multi threads") {
```

Comment:
Add a testcase for negative configured property(CARBON_DRIVER_PRUNING_MULTI_THREAD_ENABLE_FILES_COUNT)